libs/blocks/quiz/utils.js

```diff
@@ -150,7 +160,7 @@ export const storeResultInLocalStorage = (
     ),
     pageloadHash: getAnalyticsDataForLocalStorage(analyticsConfig),
   };
-  localStorage.setItem(quizKey, JSON.stringify(resultToDelegate));
+  localStorage.setItem(quizLocalKey, JSON.stringify(resultToDelegate));
```

Comment:
Since quizLocalKey isn't used anywhere else, perhaps define and init it here, within the storeResultInLocalStorage method?